### DISCUSSION TITLE
output no longer printing in new lines (missing CR/LF)

### DISCUSSION THREAD

**Discussion by bsvinay** [microsoft/vscode-jupyter#12500] [Questions and Answers] [answered]
Not sure what I did, it started printing all content in the same line. 
example:
print('a')
print('b')
expected output:
a
b
seeing:
a b

I tried reinstalling vscode, jupyter, nothing seems to help.

Can someone tell me how to fix this?

**Comment by capdevc** [comment]
I'm having the same issue. Not sure if I should open an issue or not since it's already been reported here.

Versions:
vscode: Version: 1.75.0-insider Commit: 727d2d32aea02baf7d13fa38787e8ed64dd858d0
vscode-python: v2023.1.10062310 (current pre-release)
vscode-jupyter: v2023.1.1003661022 (current pre-release)
vscode-notebook-renderers: v1.0.12

Example output:
<img width="361" alt="image" src="https://user-images.githubusercontent.com/1211248/211241259-5bb2f1fd-4d24-4fcb-8d66-8d29472975b1.png">

Active renderer for that output:
<img width="599" alt="image" src="https://user-images.githubusercontent.com/1211248/211241303-defa80d2-5706-47ab-8bdf-df2e43320494.png">

Happy to turn this into an issue or add debug logs etc if requested.

**Comment by firegate22** [comment, marked as answer]
@capdevc can confirm this as well - as temp fix an older insiders build should do it (which is for some reason not getting an update the last days, isn't it?). I would suggest to create an issue / bug report.

**edit: issue is fixed with new / current insiders build:** 
Version: 1.75.0-insider
Commit: 04b15ff231a2c3cfcdd90d90c16075a9c1e6da15
Date: 2023-01-09T05:45:37.517Z (3 hrs ago)

**Reply by capdevc** [reply]
I can confirm the fix too.. it's fixed in the current insiders build
